src/lib/banal/cuda/CudaBlock.cpp

```diff
@@ -1,4 +1,7 @@
 #include "CudaBlock.hpp"
+#include <Instruction.h>
+
+#define DYNINST_INSTRUCTION_PTR 0
```

Comment:
This is actually for DYNINST 10 compatibility, where they have changed `Insns` structure.